packages/mock-account-servicing-lib/src/setupFromSeed.ts

```diff
@@ -1,25 +1,33 @@
-import {
-  CONFIG,
-  type Config,
-  type Account,
-  type Peering
-} from './parse_config.server'
-import {
-  createAsset,
-  createPeer,
-  depositPeerLiquidity,
-  createWalletAddress,
-  createWalletAddressKey,
-  setFee,
-  depositAssetLiquidity,
-  createAutoPeer
-} from './requesters'
 import { v4 } from 'uuid'
-import { mockAccounts } from './accounts.server'
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
 import { generateJwk } from '@interledger/http-signature-utils'
-import { Asset, FeeType } from 'generated/graphql'
+import { createRequesters } from './requesters'
+import { Config, Account, Peering } from './types'
+import { Asset, FeeType } from './generated/graphql'
+import { AccountProvider } from './AccountProvider'
+
+interface SetupFromSeedOptions {
+  debug?: boolean
+}
+
+export async function setupFromSeed(
+  config: Config,
+  apolloClient: ApolloClient<NormalizedCacheObject>,
+  mockAccounts: AccountProvider,
+  options: SetupFromSeedOptions = {}
+): Promise<void> {
+  const { debug = false } = options
+  const {
+    createAsset,
+    depositAssetLiquidity,
+    setFee,
+    createPeer,
+    depositPeerLiquidity,
+    createAutoPeer,
+    createWalletAddress,
+    createWalletAddressKey
+  } = createRequesters(apolloClient, debug)
```

Comment:
This was kind of a tough line to draw between "what should be in mock-account-servicing-lib" and "what should be in the mock account servicing entites (in localenv, testenv)".

I added these requests to the lib because `setupFromSeed` depends on them and `setupFromSeed` is needed by both testenv and localenv. However, I dont think defining all requests in general from the library makes much sense (each MASE might want to call them differently for example). So I just added the ones `setupFromSeed` needed and kept them internal to this package. I did `createRequesters` instead of exporting each request individually because this is the only thing that needs them (nothing that needs to import a single request, nor are they exported from the package) and so that we don't have to pass in `apolloClient` to each one.

Reply:
I think these ones make sense to be in the library.